module/modmanager/manager.go

```diff
@@ -995,7 +1023,7 @@ func (m *module) registerResources(mgr modmaninterface.ModuleManager, logger log
 		switch {
 		case api.API.IsComponent():
 			for _, model := range models {
-				logger.Infow("registering component from module", "module", m.cfg.Name, "API", api.API, "model", model)
+				logger.Infow("Registering component from module", "module", m.cfg.Name, "API", api.API, "model", model)
```

Comment:
I know this is pre-existing, but
nit: component -> component model

Reply:
Good point; did `Registering component API and model from module` instead to be more accurate.